src/views/SpaceProposal.vue

```diff
@@ -101,15 +102,28 @@ async function loadProposal() {
 }
 
 async function loadResults() {
+  if (proposal.value.scores_state !== '') {
+    results.value = {
+      resultsByVoteBalance: proposal.value.scores,
+      resultsByStrategyScore: proposal.value.scores_by_strategy,
+      // @TODO take this value from API
+      sumOfResultsBalance: proposal.value.scores_by_strategy
+        .flat()
+        .reduce((a, b) => a + b, 0)
+    };
+    console.log(results.value.sumOfResultsBalance);
+    loadedResults.value = true;
+  }
   proposalObj.value.votes = await getProposalVotes(id);
   const resultsObj = await getResults(
     props.space,
     proposalObj.value.proposal,
     proposalObj.value.votes
   );
-  results.value = resultsObj.results;
-  votes.value = resultsObj.votes;
+  if (proposal.value.id !== ens) results.value = resultsObj.results;
```

Comment:
I don't get this one. The request is still made but results are just not used for the ENS proposal?

Reply:
The hub return empty array for votes on ENS proposal for now